license-maven-plugin/src/test/resources/styles/javapkg_style.txt

```diff
@@ -0,0 +1,16 @@
+
+/*-
```

Comment:
On Tue, 16 Aug 2016, Mathieu Carbou wrote:

> `/*-` is not really common amongst Java devs and I am pretty sure
> people would like the standard one, which is just `/*`

Really?

I don’t insist on it, it’s just best practice and to avoid some
tools complaning, but I’ll drop the hyphen-minus from the format.

bye,
//mirabilos
-- 
tarent solutions GmbH
Rochusstraße 2-4, D-53123 Bonn • http://www.tarent.de/
Tel: +49 228 54881-393 • Fax: +49 228 54881-235
HRB 5168 (AG Bonn) • USt-ID (VAT): DE122264941
Geschäftsführer: Dr. Stefan Barth, Kai Ebenrett, Boris Esser, Alexander Steeg